app/views/dashboards/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<h1 class="t-display page__heading">Dashboard</h1>
+<% @title = t('.title') %>
```

Comment:
@yhirano55 I am pretty sure we don't want to remove the heading `Dashboard` from the page.